apps/storybook/package.json

```diff
@@ -4,20 +4,30 @@
 	"license": "UNLICENSED",
 	"description": "Storybook application for documentation of our UI components",
 	"main": "src/index.js",
-	"devDependencies": {
-		"@babel/core": "^7.7.7",
-		"@lundium/components": "^0.0.1",
-		"@storybook/addon-actions": "5.3.1",
-		"@storybook/addon-info": "5.3.1",
-		"@storybook/addon-knobs": "5.3.1",
-		"@storybook/addon-options": "5.3.1",
-		"@storybook/react": "5.3.1",
-		"@storybook/theming": "5.3.1",
-		"react": "^16.12.0"
-	},
-	"dependencies": {},
 	"scripts": {
 		"build": "build-storybook -o ../../build/storybook -c .storybook",
 		"start": "start-storybook -p 8001 -c .storybook"
+	},
+	"dependencies": {
+		"react": "^16.12.0",
+		"react-dom": "^16.12.0"
+	},
+	"devDependencies": {
+		"@babel/core": "^7.8.3",
+		"@babel/preset-env": "^7.8.3",
```

Comment:
not needed, we use babel-preset-react-union (remember our discussion about `rootMode: "upward"`